### PR TITLE
Add a diff version of the proposal in governance#8

### PR DESCRIPTION
**Pull request by funnelfiasco**
Fixes guacsec/governance#8

# Description of the PR

# PR Checklist

- [ ] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [ ] All new changes are covered by tests
- [ ] If GraphQL schema is changed, `make generate` has been run
- [ ] If OpenAPI spec is changed, `make generate` has been run
- [ ] If `collectsub` protobuf has been changed, `make proto` has been run
- [ ] All CI checks are passing (tests and formatting)
- [ ] All dependent PRs have already been merged

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]

File: CONTRIBUTING.md
Comment (lumjjb) [inline CONTRIBUTING.md:47]: Require N-1 maintainer approval

**Review by jeffmendoza — APPROVED** [review]

File: CONTRIBUTING.md
Comment (lumjjb) [reply CONTRIBUTING.md:47]: I think for these changes, it would be helpful to get some additional community feedback to see if this helps other users contribute better as well. I suggest that perhaps when this gets to a place where we think its ready, we can share it in the mailing lists / community meeting.

File: CONTRIBUTING.md
Comment (funnelfiasco) [reply CONTRIBUTING.md:73]: I disagree. "Technical expertise" doesn't have to be in code (which is why I don't like calling contributions "non-technical") and reviews & authorship still apply. I'm fine with dropping the word "technical" here if it's a hangup, but I think the text applies as-is to all of the existing and proposed topic areas.

File: CONTRIBUTING.md
Comment (funnelfiasco) [reply CONTRIBUTING.md:60]: Yeah, I'm not sure what it would look like at this point. The reason I included it here is because it's a still topic area, and the ladder is special cased below. For that reason, I think it makes sense as presented, but I'm open to suggestions for alternative wording/presentation. On the other hand...

Full disclosure: I included this now because It's an area where I can help. It's not something that most folks are clamoring to help with in any project Iv'e seen, so it might be that I'm the only one who ends up with this role for a while anyway. Another approach is to leave this out and either 1. I don't contribute in this area or 2. the maintainers grant one-off triage privileges on the repos to help with this and we treat future requests on a one-off basis until we start getting a significant number of those.

File: CONTRIBUTING.md
Comment (funnelfiasco) [reply CONTRIBUTING.md:47]: That makes sense. I posted it in Slack when I opened the issue, but more attention on it wouldn't hurt. I'll put it on the agenda for the 20 June Community Meeting. If we put it off to the July meeting, we're probably bikeshedding too much.

I will say up front that I wouldn't be surprised if it doesn't move the needle much for the existing community members (although it might!). It's more of a foundation-laying step for trying to grow the community and bring in new contributors.

**Comment by funnelfiasco** [comment]
I removed the addition of a "Triage and Project Management" topic area from the proposal after a chat with Brandon. I agree that it doesn't quite fit with the rest and is both something we can handle on a separate basis and also not an area where we'll have people begging to contribute.

I'll squash these (and any subsequent) commits into one before we merge.

File: CONTRIBUTING.md
Comment (lumjjb) [reply CONTRIBUTING.md:47]: I think june meeting is fine! agree, this shouldnt block till july

File: CONTRIBUTING.md
Comment (lumjjb) [reply CONTRIBUTING.md]: I think in my mind i associate technical in a code repo as code. However, this is probably just my english interpretation of the word. I am fine either way, this was just a nit.

**Review by lumjjb — APPROVED** [review]

**Comment by lumjjb** [comment]
@pxp928 @SantiagoTorres @mlieberman85 require 2 more approvals for governance changes. PTAL

**Review by mlieberman85 — APPROVED** [review]
Lgtm

**Review by pxp928 — APPROVED** [review]
LGTM

**Comment by funnelfiasco** [comment]
This has a +1 from 5/6 maintainers and no -1s, so I think this can be merged.

**Comment by mihaimaruseac** [comment]
Needs rebase and fixing DCO in the first commit (and removing the `DO NOT MERGE` part)

**Pull request merged**
